web/src/beta/components/fields/DateTimeField/EditPanel/index.tsx

```diff
@@ -0,0 +1,116 @@
+import Button from "@reearth/beta/components/Button";
+import PanelCommon from "@reearth/beta/components/fields/CameraField/PanelCommon";
+import { useT } from "@reearth/services/i18n";
+import { styled } from "@reearth/services/theme";
+
+import TextInput from "../../common/TextInput";
+import SelectField from "../../SelectField";
+
+import useHooks from "./hooks";
+
+type Props = {
+  onChange?: (value?: string | undefined) => void;
+  onClose: () => void;
+};
+
+const EditPanel: React.FC<Props> = ({ onChange, onClose }) => {
+  const t = useT();
+
+  const {
+    date,
+    time,
+    timezones,
+    selectedTimezone,
+    onDateChange,
+    onTimeChange,
+    getUniqueTimezones,
+    onTimezoneSelect,
+    onDateTimeApply,
+  } = useHooks({ onChange });
+
+  const isButtonDisabled = date.trim() === "" || time.trim() === "";
+  const uniqueTimezones = getUniqueTimezones(timezones);
```

Comment:
You are already doing this in the hooks with `offsetFromUTC`. Just pass that from the hooks and use that instead.